schema.graphql

```diff
@@ -336,7 +336,7 @@ input CreateUserTagInput {
   name: String!
   organizationId: ID!
   parentTagId: ID
-  tagColor: String!
+  tagColor: String
```

Comment:
Isn't `tagColor` a "required" parameter? Why do we want it to be optional here?

Reply:
A [default color}(https://github.com/PalisadoesFoundation/talawa-api/blob/d0c167bb942c4778fba221c2cdd27665fc7dbf61/src/models/OrganizationTagUser.ts#L33) was added.

Color support was added [recently](https://github.com/PalisadoesFoundation/talawa-api/pull/2006), not sure exactly how colors would work for tags.

For now I have made it nullable, focusing on the implementation of basic functionalities and faster development.

Reply:
Makes sense.